### PR TITLE
new Cosmos DB ARM templates -- task 1158120

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
https://mseng.visualstudio.com/TechnicalContent/_queries/edit/1158120

### CODE REVIEW

File: 101-cosmosdb-create-arm-cassandra/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Create a Cassandra Cosmos Database Account
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-cassandra/README.md]: Please change to "Create an Azure Cosmos DB account for the Cassandra API"

File: 101-cosmosdb-create-arm-gremlin/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Create a Gremlin Cosmos Database Account
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-gremlin/README.md]: Please change to "Create an Azure Cosmos DB account for the Gremlin (Graph) API"

File: 101-cosmosdb-create-arm-gremlin/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-gremlin/metadata.json]: Please change to "Create an Azure Cosmos DB Gremlin (Graph) API account"

File: 101-cosmosdb-create-arm-gremlin/metadata.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Gremlin API", 
+   "description": "This template creates a Cosmos Database Account using the Gremlin API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-gremlin/metadata.json]: Change to "This template creates an Azure Cosmos DB Gremlin (Graph) API account."

File: 101-cosmosdb-create-arm-mongo/metadata.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Mongo API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-mongo/metadata.json]: Please change to "Create an Azure Cosmos DB MongoDB API account."

File: 101-cosmosdb-create-arm-mongo/metadata.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Mongo API", 
+   "description": "This template creates a Cosmos Database Account using the Mongo API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-mongo/metadata.json]: Please change to "This template creates an Azure Cosmos DB MongoDB API account."

File: 101-cosmosdb-create-arm-sql/README.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Create an SQL Cosmos Database Account
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-sql/README.md]: Change to "Create an Azure Cosmos DB SQL API account"

File: 101-cosmosdb-create-arm-sql/README.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-create-arm-sql%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-create-arm-sql%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates an Azure Cosmos Database Account using the SQL API.
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-sql/README.md]: Change to "This template creates an Azure Cosmos Database SQL API account."

File: 101-cosmosdb-create-arm-sql/metadata.json [hunk 9]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the SQL API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-sql/metadata.json]: Change to "Create an Azure Cosmos DB SQL API account"

File: 101-cosmosdb-create-arm-sql/metadata.json [hunk 10]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the SQL API", 
+   "description": "This template creates a Cosmos Database Account using the SQL API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-sql/metadata.json]: Change to "This template creates an Azure Cosmos DB SQL API account"

File: 101-cosmosdb-create-arm-sql/metadata.json [hunk 11]
```diff
@@ -0,0 +1,7 @@
...
+   "summary": "Create a Cosmos Database Account using the SQL API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-sql/metadata.json]: Change to "Create an Azure Cosmos DB SQL API account"

File: 101-cosmosdb-create-arm-table/README.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
+# Create a Table Cosmos Database Account
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-table/README.md]: Change to "Create an Azure Cosmos DB Table API account"

File: 101-cosmosdb-create-arm-table/README.md [hunk 13]
```diff
@@ -0,0 +1,10 @@
...
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-create-arm-table%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-create-arm-table%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates an Azure Cosmos Database Account using the Table API.
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-table/README.md]: Change to "This template creates an Azure Cosmos DB Table API account."

File: 101-cosmosdb-create-arm-table/azuredeploy.json [hunk 14]
```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "databaseAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The Cosmos database account name."
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-table/azuredeploy.json]: Change to "The Azure Cosmos DB account name."

File: 101-cosmosdb-create-arm-table/metadata.json [hunk 15]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Table API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-table/metadata.json]: Change to "Create an Azure Cosmos DB Table API account"

File: 101-cosmosdb-create-arm-table/metadata.json [hunk 16]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Table API", 
+   "description": "This template creates a Cosmos Database Account using the Table API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-table/metadata.json]: Change to "This template creates an Azure Cosmos DB Table API account"

File: 101-cosmosdb-create-arm-table/metadata.json [hunk 17]
```diff
@@ -0,0 +1,7 @@
...
+   "summary": "Create a Cosmos Database Account using the Table API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-table/metadata.json]: Change to "Create an Azure Cosmos DB Table API account"

File: 101-documentdb-account-create-multi-region-account/README.md [hunk 18]
```diff
@@ -1,4 +1,4 @@
-# Create a Multi-Region DocumentDB Database Account
+# Create a Multi-Region CosmosDB Database Account
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/README.md]: Change to "Create a Multi-region Azure Cosmos DB account"

File: 101-documentdb-account-create-multi-region-account/README.md [hunk 19]
```diff
@@ -7,9 +7,9 @@
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-This template will create a DocumentDB Database Account with the two specifed regions, the provided name, and the Offer Type set to ***Standard***.
+This template will create a CosmosDB Database Account with the two specifed regions, the provided name, and the Offer Type set to ***Standard***.
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/README.md]: Change "CosmosDB" to "Azure Cosmos DB"

File: 101-documentdb-account-create-multi-region-account/azuredeploy.json [hunk 20]
```diff
@@ -5,13 +5,13 @@
     "databaseAccountName": {
       "type": "string",
       "metadata": {
-        "description": "The DocumentDB database account name."
+        "description": "The Cosmos DB database account name."
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/azuredeploy.json]: Change "Cosmos DB database account name" to "Azure Cosmos DB account name"

File: 101-documentdb-account-create-multi-region-account/metadata.json [hunk 21]
```diff
@@ -1,7 +1,7 @@
 { 
-   "itemDisplayName": "Create a Multi-Region DocumentDB Database Account", 
-   "description": "This template creates a DocumentDB Database Account with two regions with a default consistency level of Session.", 
-   "summary": "Create a Multi-Region DocumentDB Database Account", 
+   "itemDisplayName": "Create a Multi-Region CosmosDB Database Account",
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/metadata.json]: Change to "Create a multi-region Azure Cosmos DB account"

File: 101-documentdb-account-create-multi-region-account/metadata.json [hunk 22]
```diff
@@ -1,7 +1,7 @@
 { 
-   "itemDisplayName": "Create a Multi-Region DocumentDB Database Account", 
-   "description": "This template creates a DocumentDB Database Account with two regions with a default consistency level of Session.", 
-   "summary": "Create a Multi-Region DocumentDB Database Account", 
+   "itemDisplayName": "Create a Multi-Region CosmosDB Database Account", 
+   "description": "This template creates a CosmosDB Database Account with two regions with a default consistency level of Session.",
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/metadata.json]: Change to "This template creates an Azure Cosmos DB account with two regions..."

File: 101-documentdb-account-create-multi-region-account/metadata.json [hunk 23]
```diff
@@ -1,7 +1,7 @@
 { 
-   "itemDisplayName": "Create a Multi-Region DocumentDB Database Account", 
-   "description": "This template creates a DocumentDB Database Account with two regions with a default consistency level of Session.", 
-   "summary": "Create a Multi-Region DocumentDB Database Account", 
+   "itemDisplayName": "Create a Multi-Region CosmosDB Database Account", 
+   "description": "This template creates a CosmosDB Database Account with two regions with a default consistency level of Session.", 
+   "summary": "Create a Multi-Region CosmosDB Database Account",
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/metadata.json]: Change to "Create a multi-region Azure Cosmos DB account"

File: 101-cosmosdb-create-arm-mongo/azuredeploy.parameters.json [hunk 24]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-mongo/azuredeploy.parameters.json]: Please change the name of this file to use "mongodb" not "mongo"

File: 101-cosmosdb-create-arm-mongo/metadata.json [hunk 25]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-mongo/metadata.json]: Please change the name of this file to use "mongodb" not "mongo"

File: 101-cosmosdb-create-arm-mongo/README.md [hunk 26]
```diff
@@ -0,0 +1,10 @@
+# Create a Mongo Cosmos Database Account
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-mongo/README.md]: Change the name of this file to use "mongodb" not "mongo"

File: 101-cosmosdb-create-arm-mongo/azuredeploy.json [hunk 27]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-mongo/azuredeploy.json]: Change the name of this file to use "mongodb" not "mongo"

File: 101-cosmosdb-create-arm-gremlin/metadata.json [hunk 28]
```diff
@@ -0,0 +1,7 @@
...
+   "summary": "Create a Cosmos Database Account using the Gremlin API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-gremlin/metadata.json]: Change to "Create an Azure Cosmos DB Gremlin (Graph) API account"

File: 101-cosmosdb-create-arm-gremlin/azuredeploy.json [hunk 29]
```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "databaseAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The Cosmos database account name."
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-gremlin/azuredeploy.json]: Change to "The Azure Cosmos DB account name."

File: 101-cosmosdb-create-arm-cassandra/metadata.json [hunk 30]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Apache Cassandra API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-cassandra/metadata.json]: Change to "Create an Azure Cosmos DB Cassandra API account"

File: 101-cosmosdb-create-arm-cassandra/metadata.json [hunk 31]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Apache Cassandra API", 
+   "description": "This template creates a Cosmos Database Account using the Apache Cassandra API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-cassandra/metadata.json]: Change to "This template creates an Azure Cosmos DB Cassandra API account"

File: 101-cosmosdb-create-arm-cassandra/metadata.json [hunk 32]
```diff
@@ -0,0 +1,7 @@
...
+   "summary": "Create a Cosmos Database Account using the Apache Cassandra API",
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-cassandra/metadata.json]: Change to "Create an Azure Cosmos DB Cassandra API account"

File: 101-cosmosdb-create-arm-cassandra/azuredeploy.json [hunk 33]
```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "databaseAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "The Cosmos database account name."
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-cassandra/azuredeploy.json]: Change to "The Azure Cosmos DB account name."

File: 101-cosmosdb-create-arm-cassandra/README.md [hunk 34]
```diff
@@ -0,0 +1,10 @@
...
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-create-arm-cassandra%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-create-arm-cassandra%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates an Azure Cosmos Database Account using the Cassandra API.
```
Comment (mimig1) [inline 101-cosmosdb-create-arm-cassandra/README.md]: Change to "This template creates an Azure Cosmos DB Cassandra API account."

File: 101-documentdb-account-create-multi-region-account/README.md [hunk 35]
```diff
@@ -7,9 +7,9 @@
...
 
 By not setting the optional Default Consistency Level parameter, the account will be created with the default consistency level of ***Session***.
```
Comment (mimig1) [inline 101-documentdb-account-create-multi-region-account/README.md]: This additional content from the original readme should be included in all of the other readmes as well (as much as is applicable).

**Comment by mimig1** [comment]
These changes all look good to me. @WilliamAntonRohm - Please sign off when ready.

**Comment by WilliamAntonRohm** [comment]
#sign-off

**Comment by mimig1** [comment]
@bmoore-msft - Do you merge PRs in this repo? Can you review/merge this one? Thanks.

**Comment by WilliamAntonRohm** [comment]
these templates are being replaced by a single combined template

**Comment by WilliamAntonRohm** [comment]
starting over

**Pull request closed without merging**